CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## 0.52.3      2023-11-30
+
+* Reduce frequency of collector log generation
+  - Previously, real-time snapshots would generate log lines every 10 seconds,
+    which made errors hard to find
+  - Now, a single log line is printed once a minute with a summary of snapshots
+    submitted
+  - Note that `--verbose` will still log every snapshot as it's submitted
+* OpenTelemetry integration:
+  - Support `pganalyze` tracestate to set start time of the span
+  - Start time can be specified with `t` member key, the Unix time in second,
+    allowing decimals to present up to nano seconds
```

Comment:
I think the current wording is a little unclear; here is one alternative suggestion:
```suggestion
  - Start time can be specified with `t` member key as Unix time in seconds,
    with decimals to specify precision down to nano seconds
```